src/Core/maps/src/Platform/iOS/MapPool.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+using MapKit;
+
+namespace Microsoft.Maui.Maps.Platform
+{
+	internal class MapPool
```

Comment:
Do we still want this? Anyone knows why we used this "pool" ?

Reply:
We still support iOS 10 so I'm guessing the answer here is yes

https://github.com/xamarin/Xamarin.Forms/pull/467

@hartez thoughts?

Reply:
Yeah, until we can drop support for 10 I think we need to keep it.